griptape/common/prompt_stack/contents/text_prompt_stack_content.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from attrs import define, field
+from collections.abc import Sequence
+
+from griptape.artifacts import TextArtifact
+from griptape.common import BasePromptStackContent, BaseDeltaPromptStackContent, TextDeltaPromptStackContent
+
+
+@define
+class TextPromptStackContent(BasePromptStackContent):
+    artifact: TextArtifact = field(metadata={"serializable": True})
+
+    @classmethod
+    def from_deltas(cls, deltas: Sequence[BaseDeltaPromptStackContent]) -> TextPromptStackContent:
+        text_deltas = [delta for delta in deltas if isinstance(delta, TextDeltaPromptStackContent)]
```

Comment:
should this silently fail if its not the right type?

Reply:
Not sure I follow -- this will not silently fail as it currently is.